internal/receiver/smartagentreceiver/receiver.go

```diff
@@ -17,6 +17,8 @@ package smartagentreceiver
 import (
 	"context"
 	"fmt"
+	"github.com/signalfx/splunk-otel-collector/internal/zaputil"
+	"github.com/sirupsen/logrus"
```

Comment:
`make lint` would reveal these are in undesired groups.  Also think you want to run `go fmt`.